minpy/array_variants/mxnet/mxnet_core.py

```diff
@@ -101,7 +101,8 @@ def register_primitives(reg, prim_wrapper):
     """ Register primitives in mxnet package """
     mxnet_wrapper.wrap_namespace(mx.nd.__dict__, reg, prim_wrapper)
     # Additional primitives due to naming issues in MXNet.
-    reg.register('reshape', prim_wrapper(NDArray.reshape))
+    if not 'reshape' in mx.nd.__dict__.keys():
```

Comment:
How about just changing `reg.register` function? For example, add an `if` statement and throw a Warning in the log if exist?